acl/acl-lib.pl

```diff
@@ -1690,10 +1690,17 @@ sub encrypt_password
 	return &encrypt_sha512($pass, $salt);
 	}
 else {
-	# Use Unix DES
-	&seed_random();
-	$salt ||= chr(int(rand(26))+65).chr(int(rand(26))+65);
-	return &unix_crypt($pass, $salt);
```

Comment:
Isn't this change going to break any validation of existing passwords that are encrypted with old-style crypt() ?

Reply:
No, because it's not deleted. Please check slightly below. It will use system default and if module useradmin not available it will fall-back to old DES.

Reply:
Validation will happen in `validate_password` sub in `md5-lib.pl`, which already supports latest formats and been merged few days ago.